src/nuis/measurement/ProjectionHelper.cxx

```diff
@@ -177,27 +177,22 @@ Record::Record(std::string iname,
     }
 }
 
-Record::~Record() {
+ProjectionHelper::~Projection() {
 }
 
-void Record::Reset() {
-    ResetTally();
+void ProjectionHelper::Reset() {
```

Comment:
don't love having 2 names for the same thing, maybe you envision that this will do something else later.